app/helpers/dashboard_helper.rb

```diff
@@ -22,8 +22,8 @@ def format_period(period, value)
     period == :month ? value.strftime("%b %Y") : quarter_string(value)
   end
 
-  def analytics_totals(analytics, metric, period)
-    dash_if_zero(analytics.sum { |_, row| row.dig(metric, period) || 0 })
```

Comment:
~Removing dash_if_zero will affect the dashboard views. Is this the required behavior?~

Saw the usage below.